allensdk/test/brain_observatory/behavior/test_running_processing.py

```diff
@@ -71,21 +71,36 @@ def test_get_running_df(running_data, timestamps, lowpass):
         assert np.count_nonzero(np.isnan(actual["speed"])) == 0
 
 
-def test_get_running_df_one_fewer_timestamp_check_warning():
-    with pytest.warns():
+@pytest.mark.parametrize(
+    "lowpass", [True, False]
+)
+def test_get_running_df_one_fewer_timestamp_check_warning(running_data,
+                                                          timestamps,
+                                                          lowpass):
+    with pytest.warns(UserWarning):
```

Comment:
For this, I would do:

`with pytest.warns(UserWarning) as w:`

And then test that the captured warning msg string starts with: `"Time array is 1 value shorter than encoder array."`